wire-ios-sync-engine/Source/Use cases/OneOnOneConversationCreationStatusUseCase.swift

```diff
@@ -0,0 +1,104 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireDataModel
+
+// sourcery: AutoMockable
+public protocol OneOnOneConversationCreationStatusUseCaseProtocol {
```

Comment:
suggestion: add a doc comment explaining what the use case does.

Reply:
Also, it may be nice to give it a name that describes the function it performs, such as `GetOneOnOneConversationStatusUseCaseProtocol`

Reply:
updated https://github.com/wireapp/wire-ios/pull/1321/commits/5bf1d89609b0ea0c3c77278eb4fd6aacc0331a89